test/src/integration/tendermint.test.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright 2018 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import CodeChain from "../helper/spawn";
+import {
+    aliceAddress,
+    aliceSecret,
+    faucetAddress,
+    validator0Address,
+    validator1Address,
+    validator2Address,
+    validator3Address
+} from "../helper/constants";
+import { U256 } from "codechain-sdk/lib/core/U256";
+import { PlatformAddress } from "codechain-sdk/lib/core/classes";
+
+const testSkippedInTravis = process.env.TRAVIS ? test.skip : test;
+
+describe("Tendermint ", () => {
```

Comment:
It seems that all tests are skipped in Travis.
Please make the describe be skipped.